generator/Cargo.toml

```diff
@@ -6,12 +6,16 @@ license = "MIT"
 edition = "2018"
 
 [dependencies]
-cargo_metadata = "0.13"
-semver = "0.11"
-platforms = "1.1"
+cargo_metadata = "0.14"
+semver = "1.0"
+platforms = "2.0"
+# Bitflags is pulled in by clap, but 1.3 bumps the MSRV to 1.46,
+# so we ensure cargo doesn't update beyond 1.2 by specifying it 
+# as a direct dependency.
+bitflags = "~1.2.1"
```

Comment:
I don't think we should do that.
We should bump our own MSRV, or keep the old version in the Cargo.lock

Reply:
Okay, I dropped that commit.